man/man5/zfs-module-parameters.5

```diff
@@ -537,6 +537,18 @@ Min time before an active prefetch stream can be reclaimed
 Default value: \fB2\fR.
 .RE
 
+.sp
+.ne 2
+.na
+\fBzfs_abd_scatter_min_size\fR (uint)
+.ad
+.RS 12n
+This is the minimum allocation size that will use scatter (page-based)
+ABD's.  Smaller allocations will use linear ABD's.
+.sp
+Default value: \fB1536\fR (512B and 1KB allocations will be linear).
```

Comment:
Nit - This is inclusive of 1.5K

Reply:
1.5K is the smallest size that will use scatter. The only block sizes smaller than that are 512B and 1K. So I believe that the description here is accurate.